### PR TITLE
custom fields improvements

### PR DESCRIPTION
**Pull request by ntarocco**

### CODE REVIEW

File: docs/develop/topics/custom_fields.md [hunk 1]
```diff
@@ -154,9 +152,9 @@ _("Program") }}: {{value.get("program", "Unknown")}}) {{ ", " if not loop.last
 }} {% endfor %}
 ```
 
-## Applying configuration
+## Configuration
 
-We have defined our field, so now we need to add it to the `RDM_CUSTOM_FIELDS` and `RDM_CUSTOM_FIELDS_UI` configuration:
+Finally, we need to add the new field to the `RDM_CUSTOM_FIELDS` and `RDM_CUSTOM_FIELDS_UI` configuration:
```
Comment (ntarocco) [inline docs/develop/topics/custom_fields.md:157]: @zzacharo here I am missing the import. How do I import `ExperimentsCF`?

Comment (ppanero) [reply docs/develop/topics/custom_fields.md:157]: There is no import, the field was implemented above (line 32). So there is the assumption that is on the same file (invenio.cfg) or the user knows where they put it (e.g. their own module). True that we should specify this.

**Review by ppanero — APPROVED** [review]

**Comment by ppanero** [comment]
Thanks for the review/changes!

File: docs/develop/topics/custom_fields.md [hunk 2]
```diff
@@ -21,18 +22,15 @@ To satisfy the the above scenario we will implement a new field `experiments` wh
 }
 ```
 
-This will give us the opportunity to add the experiments a record can belong to, and more information about them.
-
-## Implement a new custom field
-
-We will start by defining a new custom field type.
+Let's define a new custom field type.
```
Comment (zzacharo) [inline docs/develop/topics/custom_fields.md:25]: Maybe we can add here a note "For convenience, we will define our new custom field type in the `my-site/invenio.cfg` where the application config resides."

**Review by zzacharo — APPROVED** [review]

**Comment by ppanero** [comment]
@ntarocco @zzacharo added that line!

**Pull request merged**